Source/Fuse.Maps/MapView.uno

```diff
@@ -219,6 +240,16 @@ namespace Fuse.Controls
 			UpdateMarkersNextFrame();
 		}
 
+		void OnPolylineAdded(MapPolyline polyline)
+		{
+			UpdateMarkersNextFrame();
+		}
+
+		void OnPolylineRemoved(MapPolyline polyline)
+		{
+			UpdateMarkersNextFrame();
+		}
```

Comment:
Those methods are provisioning. At the time they were written it was unclear if separate handling would be needed for the two events dependent on the platform.